content/docs/user-guide/gems/reference/multiplayer/multiplayer-gem/overview.md

```diff
@@ -45,6 +45,32 @@ Then, clients update themselves from a delta by:
 
 After, the AzNetworking layer becomes aware that the world was updated. This occurs automatically through ACK vector replication, and does not require any further action from the client.
 
+### Types of network properties
+
+Network properties have two important fields: `ReplicateFrom` and `ReplicateTo`. 
+Together, these defines which role can replicate to which role. 
+Properties can replicate *from* Authority and Autonomous roles only because they have write-access. 
+Properties can replicate *to* all of the roles because all roles have read-access. 
+This structure makes sense because roles with only read-access simply receive information; they don't have any data to write or replicate. 
+
+| Field | Description | Values | 
+| - | - | - |
+| `ReplicateFrom` | Tells the network which role can send information about the changes that were made to this property. | Authority, Autonomous | 
+| `ReplicateTo` | Tells the network which role can receive information about the changes that were made to this property. | Server, Authority, Autonomous, Client | 
+
+
+The following are four types of networking properties with a valid "from and to" relationship that you can define, and their possible use cases:
+
+- **Authority-to-Client**: For handling client, or "simulated", properties.
+
+- **Authority-to-Autonomous**: For handling autonomous-only properties.
+
+- **Authority-to-Server**: For handling host migration.
+
+- **Autonomous-to-Authority**: For gathering information about client metrics, such as monitoring the health of clients, and so on.
+
+For networking properties that replicate from Authority, a replication hierarchy applies. The replicate-to rules trickle up the hierarchy in the following way: An Authority-to-Client replication also replicates to Autonomous and Server roles. An Authority-to-Autonomous replication also replicates to the Server role. Finally, an Authority-to-Server replication only replicates to the Server.
```

Comment:
_Authority-to-Autonomous replication also replicates to the Server role_
Ah, nice. I didn't know that. Is that so if the Authority ever migrates to the other server, the other server has the right property information?

Reply:
That's my understanding! @kberg-amzn spoke about it in a O3DCon talk -- can you confirm this is how the hierarchy behaves and the reason behind it?

Reply:
I've been in that code recently and confirm this is true.

Reply:
Thanks for the feedback and confirmation! I've made a small change to include the reasoning AMZN-Gene mentioned.